cdap-api/src/main/java/co/cask/cdap/api/workflow/NodeValueEntry.java

```diff
@@ -18,14 +18,14 @@
 
 /**
  * Multiple nodes in the Workflow can add the same key to the {@link WorkflowToken}.
- * This class provides a mapping from node name to the value which was set for the
+ * This class provides a mapping from node name to the {@link Value} which was set for the
  * specific key.
  */
 public final class NodeValueEntry {
```

Comment:
Make sense to call it `NodeValue`, because it's a class containing node name and value.